### ISSUE TITLE
SecurityContext is deprecated since version 2.4 and will be removed in version 3.0

### ISSUE THREAD

**Issue by TheRatG** [sonata-project/SonataUserBundle#598]
Good day. 
Since we started use symfony 2.7, we get a lot of messages like this.

```
#vendor/sonata-project/user-bundle/Controller/AdminSecurityController.php(58)

The Symfony\Component\Form\Extension\Csrf\CsrfProvider\CsrfTokenManagerAdapter::generateCsrfToken method is deprecated since version 2.4 and will be removed in version 3.0. Use the Symfony\Component\Security\Csrf\CsrfTokenManager class instead.
```

**Comment by Soullivaneuh** [comment]
Duplicate of #568.

Please search on existing ticket before starting a new one, thanks.

**Comment by TheRatG** [comment]
Thanks :smile:
